bigbluebutton-html5/imports/api/cursor/server/handlers/cursorUpdate.js

```diff
@@ -1,5 +1,22 @@
 import { check } from 'meteor/check';
-import updateCursor from '../modifiers/updateCursor';
+import { CursorStreamer } from '/imports/api/cursor';
+
+let cursorQueue = [];
+let cursorRecieverIsRunning = false;
+
+const proccess = () => {
+  if (!Object.keys(cursorQueue).length) {
+    cursorRecieverIsRunning = false;
+    return;
+  }
+  cursorRecieverIsRunning = true;
+  Object.keys(cursorQueue).forEach(meetingId => {
+    CursorStreamer.emit('message', { meetingId, cursors: cursorQueue[meetingId] });
+  });
+  cursorQueue = {};
+
+  Meteor.setTimeout(proccess, 30);
```

Comment:
This batch interval needs to be a const as well, similarly to the one in `whiteboardSend.js`.